update_rate: 15 # Hz
encoder_ticks: 4096
device_name: "/dev/ttyUSB0"

pitch:
  id: 1
  offset: 3.14
  direction: 1
  limits: [-0.785, 0.785]

yaw:
  id: 2
  offset: 3.14
  direction: -1
  limits: [-0.785, 0.785]